File: lessons/lesson.23/morelanguages/car.cs
using System;

class Car {
    public string Make;
    public string Model;
    public int Year;
    public int Speed;

    public Car(string make, string model, int year) {
        Make = make;
        Model = model;
        Year = year;
        Speed = 0;
    }

    public void Accelerate(int increment) {
        Speed += increment;
    }

    public void Brake(int decrement) {
        Speed -= decrement;
    }

    static void Main(string[] args) {
        Car myCar = new Car("Toyota", "Corolla", 2023);
        Console.WriteLine($"My car: {myCar.Year} {myCar.Make} {myCar.Model}");

        myCar.Accelerate(20);
        Console.WriteLine($"Current speed: {myCar.Speed} mph");

        myCar.Brake(5);
        Console.WriteLine($"Current speed after braking: {myCar.Speed} mph");
    }
}

File: lessons/lesson.23/morelanguages/car.py
class Car:
    def __init__(self, make, model, year):
        self.make = make
        self.model = model
        self.year = year
        self.speed = 0

    def accelerate(self, increment):
        self.speed += increment

    def brake(self, decrement):
        self.speed -= decrement

# Creating an instance of Car
my_car = Car("Toyota", "Corolla", 2023)
print(f"My car: {my_car.year} {my_car.make} {my_car.model}")

my_car.accelerate(20)
print(f"Current speed: {my_car.speed} mph")

my_car.brake(5)
print(f"Current speed after braking: {my_car.speed} mph")
